components/mpas-albany-landice/bld/namelist_files/namelist_defaults_mali.xml

```diff
@@ -125,7 +125,7 @@
 <config_time_integration>'forward_euler'</config_time_integration>
 <config_rk_order>2</config_rk_order>
 <config_rk3_stages>3</config_rk3_stages>
-<config_adaptive_timestep>.false.</config_adaptive_timestep>
+<config_adaptive_timestep>.true.</config_adaptive_timestep>
```

Comment:
I'm wary of turning this on by default without pretty extensive testing first, but I guess that can be part of this PR.